README.md

```diff
@@ -223,6 +223,12 @@ An example of a three server deployment is shown below.  The first server holds
       node.data: true
       node.master: false
       bootstrap.memory_lock: false
+    es_scripts: false
+    es_templates: false
+    es_version_lock: false
+    ansible_user: ansible
```

Comment:
Same question for this one, except this will actually causes breakages for anyone not using the ansible user for ssh.